composer.json

```diff
@@ -3,17 +3,17 @@
     "description": "Tool for functional testing of Keboola Connection components",
     "license": "MIT",
     "require": {
-        "PHP": "^7.1",
-        "keboola/php-temp": "^1.0",
-        "phpunit/phpunit": "^7.0",
```

Comment:
Je tam hromada BC breaků:
https://phpunit.de/announcements/phpunit-9.html

Nebude to něčemu vadit?

Reply:
Resp. bude to vadit tomu, že když někde updatneš datadirtesty tak pak musíš i v testech té věci vyřešit ty BC breaky

Reply:
jj vydal bych to jako novou major verzi s tím že se to bude muset pořešit kde se bude používat nová verze a db-ex stejně datadir testy nepoužívájí